‘async’ is a reserved word in Python >= 3.7

flake8 testing of https://github.com/perone/medicaltorch on Python 3.7.0
 
$ ___flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./examples/gmchallenge_unet.py:107:42: E999 SyntaxError: invalid syntax
            var_gt = gt_samples.cuda(async=True)
                                         ^
./medicaltorch/datasets.py:479:16: F821 undefined name 're'
            if re.search('[SaUO]', elem.dtype.str) is not None:
               ^
./medicaltorch/transforms.py:26:36: F821 undefined name 'img'
            img = t.undo_transform(img)
                                   ^
1     E999 SyntaxError: invalid syntax
2     F821 undefined name 're'
```

--- a/examples/gmchallenge_unet.py
+++ b/examples/gmchallenge_unet.py
@@ -104,7 +104,7 @@ def run_main():
             input_samples, gt_samples = batch["input"], batch["gt"]
 
             var_input = input_samples.cuda()
-            var_gt = gt_samples.cuda(async=True)
+            var_gt = gt_samples.cuda(non_blocking=True)
 
             preds = model(var_input)
 

